### ISSUE TITLE
ID Number search with "My Team Requests" is not working as expected for Intake User

### ISSUE THREAD

**Issue by divyav-aot** [bcgov/foi-flow#2150]
**Describe the bug in current situation**
ID Number search with "My Team Requests" is not working as expected for Intake User. This is happening only for Redirect and Intake in progress states.

**Link bug to the User Story**

**Impact of this bug**
Describe the impact, i.e. what the impact is, and number of users impacted. 

**Chance of Occurring (high/medium/low/very low)**

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**

**Steps to Reproduce**
1. Go to Intake queue
2. Click on My Team's Request
3. Search for any request which is in Redirect/Intake in progress

**Actual/ observed behaviour/ results**
The request is not listed

**Expected behaviour**
The request should list

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

**Comment by lmullane** [comment]
@richard-aot, can I confirm that bug was this:

GIVEN a request an Intake User is on his or her queue
AND a request is in Intake In Progress or Redirect
WHEN the user searches for the request by the AXIS ID Number
THEN no results will be listed

If so, all fixed:)

**Comment by richard-aot** [comment]
@lmullane , yes, the bug was what you just described. If you search now, the redirect and intake in progress requests should show. Thanks.

**Comment by lmullane** [comment]
Thanks for the confirmation and the fix @richard-aot 

The bug is fixed.
